src/components/SharedComponents/ViewWrapper.tsx

```diff
@@ -1,9 +1,9 @@
 import classnames from "classnames";
 import { Body1 } from "components/SharedComponents";
-import { SafeAreaView } from "components/styledComponents";
 import type { PropsWithChildren } from "react";
 import * as React from "react";
-import { StatusBar } from "react-native";
+import { StatusBar, View } from "react-native";
```

Comment:
We should import View from `styledComponents` here.

Reply:
fixed